bonus.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"github.com/eoscanada/eos-go"
+	"github.com/eoscanada/eos-go/ecc"
+	"strconv"
+)
+
+type PlayerStats struct {
+	Player          string    `json:"player"`
+	SessionsCreated uint64    `json:"sessions_created"`
+	VolumeReal      eos.Asset `json:"volume_real"`
+	VolumeBonus     eos.Asset `json:"volume_bonus"`
+	ProfitReal      eos.Asset `json:"profit_real"`
+	ProfitBonus     eos.Asset `json:"profit_bonus"`
+}
+
+type ConvertBonusData struct {
+	Name eos.AccountName `json:"name"`
+	Memo string          `json:"memo"`
+}
+
+func (app *App) getBonusPlayers() ([]PlayerStats, error) {
+	resp, err := app.bcAPI.GetTableRows(eos.GetTableRowsRequest{
+		Code:  string(app.BlockChain.CasinoAccountName),
+		Scope: string(app.BlockChain.CasinoAccountName),
+		Table: "playerstats",
+		Limit: 0,
+		JSON:  true,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	var playerStats []PlayerStats
+
+	err = resp.JSONToStructs(&playerStats)
+	if err != nil {
+		return nil, err
+	}
+
+	return playerStats, nil
+}
+
+func (app *App) convertBonus(player string, force bool) error {
+	if !force {
+		meetRequirements, err := app.meetRequirements(player)
```

Comment:
can we change the signature to 
`PlayerMeetsRequirements(player string, bcAPI *eos.API) (bool, error)`?
not clear what `app.meetRequiremens` means without going into implementation

Reply:
ok